functional_tests.go

```diff
@@ -379,9 +411,9 @@ func testMakeBucketRegions() {
 		failureLog(function, args, startTime, "", "MakeBucket failed", err).Fatal()
 	}
 
-	// Remove the newly created bucket.
-	if err = c.RemoveBucket(bucketName + ".withperiod"); err != nil {
-		failureLog(function, args, startTime, "", "Remove bucket failed", err).Fatal()
+	// cleanup
```

Comment:
Add more descriptive comment rather than place holder like `cleanup`

Reply:
Agreed. Updated the comments